packages/terra-action-footer/src/ActionFooter.module.scss

```diff
@@ -17,5 +17,6 @@
   .start-actions,
   .end-actions {
     align-self: center;
+    flex: 0 1 auto; //Needed for IE 10 to wrap things properly
```

Comment:
This can also be just `flex: 1` to make all the flexible items to be the same length, regardless of its content. 
reference : https://www.w3schools.com/cssref/css3_pr_flex.asp

Reply:
I tried testing this, and it changed the layout of the end section of the action footer. It aligns the end content to the middle of the footer in both chrome and IE, as opposed to right aligning it.

I found some sources [1][2] which suggest that `flex: 1` is the same as `flex: 1 1 0`. Which is what I am seeing in chrome:
![Screen Shot 2020-06-04 at 1 11 24 PM](https://user-images.githubusercontent.com/31047566/83795859-b7931980-a665-11ea-9cd3-6b1768f6449c.png)

[1]https://stackoverflow.com/questions/37386244/what-does-flex-1-mean
[2]https://www.w3.org/TR/css-flexbox-1/#flex-common

Reply:
@supreethmr -- setting this to `flex: 1` would actually be counter-productive since it changes the component _away_ from modern browser default/initial behavior, which is what we want to set explicitly for IE10. As @JacksonJN mentions, `flex: 1` is the same as `flex: <positive-number>` which equates to `flex: <positive-number> 1 0`.

Specifying the full initial value: `flex: 0 1 auto` is the best fix to "reset" IE10 to shim it to match the default `flex` behavior in all the other supported browsers. For more reasons why - read up on the history of IE10 implementing a very early draft spec for Flexbox in 2012 https://github.com/cerner/terra-core/issues/3009#issuecomment-640904254